packages/yoga/src/Card/native/EventCard/EventCard.jsx

```diff
@@ -9,15 +9,17 @@ import Text from '../../../Text';
 import Box from '../../../Box';
 import Button from '../../../Button';
 
-const { spacing, radii, lineHeights } = theme;
+const { spacing, radii, lineHeights, colors, borders } = theme;
 
-const Event = styled(Card)`
+const Event = styled(Card).attrs({ hasShadow: false })`
   flex-direction: row;
   height: 104px;
   border-radius: ${radii.regular}px;
   padding: ${spacing.zero};
   width: 280px;
   margin-right: ${spacing.xxsmall};
+  border: ${borders.small}px solid ${colors.elements.lineAndBorders};
```

Comment:
Currently, the `colors.elements.lineAndBorders` color does not match the value displayed in Figma (#D7D7E0 vs #EFECE7), but I decided to use the token listed there. Is the `lineAndBorders` color going to be updated to `#EFECE7` ?

<img width="674" alt="image" src="https://github.com/gympass/yoga/assets/60697831/3a40639b-d164-47e0-bea4-cec9806d1fcb">